resources/docker/ssh/sshd_config_password

```diff
@@ -0,0 +1,126 @@
+
+# This is the sshd server system-wide configuration file.  See
```

Comment:
It's better to just do it like in the buildfs tests - use sed to change the default ssh config. Just by looking at this file I don't know what changes were made compared to the defaults.

Reply:
I believe this is actually clearer than sed. I'm leaving it as is